### PR TITLE
updated Torsteins mlst container, as results were differing with PubMLST

### PR DESCRIPTION
**Pull request by mattheww95**
Updated the default MLST container and added change log message. The container update for alternate registries is not made yet, so this PR does not address it.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### `Updated`
+
+- Updated TSeemann's MLST default container. See [PR 97](https://github.com/phac-nml/mikrokondo/pull/97)
```
Comment (apetkau) [inline CHANGELOG.md:10]: Can you mention which version you update to.

**Review by apetkau — APPROVED** [review]
Looks good. Thanks so much. Just one change and you can merge.

File: CHANGELOG.md (hunk 1 above)
Comment (mattheww95) [reply CHANGELOG.md]: Gladly! fixed in: https://github.com/phac-nml/mikrokondo/pull/98/commits/5ac1399fb1bbee0d7e22db2eb5eb052ca384612f

Comment (apetkau) [reply CHANGELOG.md]: Thanks so much.

**Pull request merged**